survey/SHACL_SurveyServiceOffering.ttl

```diff
@@ -0,0 +1,67 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix surveyonto: <http://semanticweb.org/metadatasurveyontology/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+surveyonto:SurveyServiceOfferingShape
+	a sh:NodeShape ; 
+	sh:targetClass surveyonto:SurveyServiceOffering ; 
+	sh:property [
+		sh:path surveyonto:survey_id ; 
+		sh:datatype xsd:string ; 
+		skos:example "'Example ID'" ;
+		sh:minCount 1 ; 
+		sh:message "Missing survey ID." ;
+		sh:name "Survey ID" ;
+	] ; 
+	sh:property [
+		sh:path surveyonto:survey_title ; 
+		sh:datatype xsd:string ; 
+		skos:example "'Example Title of Survey'" ;
+		sh:minCount 1 ;
+		sh:message "Missing Survey Title." ;
+		sh:name "Survey title" ;
+	] ; 
+	sh:property [
+		sh:path surveyonto:survey_description ; 
+		sh:datatype xsd:string ; 
+		skos:example "'Example Description of Survey'" ; 
+		sh:minCount 1 ;
+		sh:message "Missing survey description." ;
+		sh:name "Survey description" ;
+	] ;
+	sh:property [
```

Comment:
Will be there in ServiceAccessPoint in the gx vocabulary. you can leave it out here.